packages/presentation/lib/base/bloc_screen.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:presentation/base/bloc.dart';
+
+abstract class BlocScreenState<BS extends StatefulWidget, B extends Bloc>
+    extends State<BS> with WidgetsBindingObserver {
```

Comment:
i think, that you won't use WidgetsBindingObserver, so you can remove WidgetsBindingObserver